docs/7.0. Measurement_units_guidelines.md

```diff
@@ -0,0 +1,32 @@
+# Measurement units guidelines
+
+_(c) AMWA 2018, CC Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)_
+
+This document describes the guidelines on how to use measurement units when defining a `number/measurement` event_type.
+
+Other sections can be accessed from the [Overview](1.0.%20Overview.md).
+
+## 1. Introduction
+
+The aim of this document is to create a set of guidelines on how to use measurement units in the Event & Tally specification with the aim of increasing compatibility between implementations.
+
+## 2. Guidelines
+
+### 2.1. Case sensitive behaviour
+
+Units of measurement can be case sensitive and this behaviour is allowed.
```

Comment:
I think we should say MUST instead of "can", as otherwise you cannot distinguish some values (e.g. mega and milli; Siemens and second etc.)